.github/workflows/ci.yml

```diff
@@ -3,41 +3,40 @@ name: CI
 on: push
 
 jobs:
-    build:
-
-        runs-on: ubuntu-latest
-        strategy:
-            matrix:
-                php-versions: ['7.4']
-        name: PHP ${{ matrix.php-versions }}
-        steps:
-            - uses: actions/checkout@v2
-
-            - name: Install PHP
-              uses: shivammathur/setup-php@v2
-              with:
-                  php-version: ${{ matrix.php-versions }}
-                  extensions: intl, bcmath
-                  tools: composer:v1
-
-            - name: Check PHP Version
-              run: php -v
-
-            - name: Validate composer.json and composer.lock
-              run: composer validate
-
-            - name: Cache Composer packages
-              id: composer-cache
-              uses: actions/cache@v2
-              with:
-                  path: vendor
-                  key: ${{ runner.os }}-php-${{ hashFiles('**/composer.lock') }}
-                  restore-keys: |
-                      ${{ runner.os }}-php-
-
-            - name: Install dependencies
-              if: steps.composer-cache.outputs.cache-hit != 'true'
-              run: composer install --no-progress --no-suggest
-
-            - name: Run CI checks
-              run: composer ci
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-versions: ["7.4"]
+    name: PHP ${{ matrix.php-versions }}
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
Kinda confusing since only 2 things changed If i looked correctly.😅

Reply:
Aha yes, it must be my editor who formatted it